criu/cgroup.c

```diff
@@ -1138,17 +1147,22 @@ static int prepare_cgns(CgSetEntry *se)
 
 			pr_info("setting cgns prefix to %s\n", ce->path);
 			snprintf(aux + aux_off, sizeof(aux) - aux_off, "/%s/tasks", ce->path);
+			args = xmalloc(sizeof(struct task_id_path));
+			args->id = pid;
+			memcpy(args->path, aux, sizeof(aux));
```

Comment:
There's probably no need to copy whole buffer (which is probably way longer then the string inside aux). There's probably a better function to copy strings.